engine/battle/stats_functions.asm

```diff
@@ -444,12 +444,10 @@ SetExpAllFlags:
 .gainExpFlagsLoop
 	ld a, [hli]
 	or [hl] ; is mon's HP 0?
-	jp z, .setzeroexpflag
+	jp z, .setnextexpflag
 	scf
-	rl b
-	jp .nextmonforexpall
-.setzeroexpflag
-	sla b
+.setnextexpflag
```

Comment:
There we go. Now in-line comments are working. Anyway, assumes the party is always 6.

Reply:
Thanks, for the correction I fixed the issue in the latest commit.